### ISSUE TITLE
bug: capacitor/clipboard only gets the first line of the text of a clipboard

### ISSUE THREAD

**Issue by apuchitnis** [ionic-team/capacitor#6736]
## Bug Report

I have code like the following:
```
Clipboard.read()
          .then(({ value: value_, type }) => {
            console.log('### got clipboard value:\n', value_, type);
```

If the following text is in the clipboard, it reads it in it's entireity:
```
Good Girl, Bad Blood: The Sequel to A Good Girl's Guide to Murder https://a.co/d/cUvlnDs
```

If the following text is in the clipboard, it only reads the first line:
```
Good Girl, Bad Blood: The Sequel to A Good Girl's Guide to Murder
https://a.co/d/cUvlnDs
```
ie only `Good Girl, Bad Blood: The Sequel to A Good Girl's Guide to Murder` is returned.

### Capacitor Version

```
💊   Capacitor Doctor  💊 

Latest Dependencies:

  @capacitor/cli: 5.2.1
  @capacitor/core: 5.2.1
  @capacitor/android: 5.2.1
  @capacitor/ios: 5.2.1

Installed Dependencies:

  @capacitor/cli: 4.8.1
  @capacitor/core: 4.8.1
  @capacitor/android: 4.8.1
  @capacitor/ios: 4.8.1

[success] iOS looking great! 👌
[success] Android looking great! 👌```

### Platform(s)
<!--
List the platforms that this bug affects.
-->

ios


### Current Behavior
<!--
Describe how the bug manifests. Be specific.
-->
I get the text:
```
Good Girl, Bad Blood: The Sequel to A Good Girl's Guide to Murder
```


### Expected Behavior
<!--
Describe what the behavior should be.
-->

I get the full text:
```
Good Girl, Bad Blood: The Sequel to A Good Girl's Guide to Murder
https://a.co/d/cUvlnDs
```

### Code Reproduction
<!--
To isolate the cause of the problem, we ask you to provide a minimal sample application that demonstrates the issue.
For full instructions, see: https://github.com/ionic-team/capacitor/blob/HEAD/CONTRIBUTING.md#creating-a-code-reproduction
-->

```
Clipboard.read()
          .then(({ value: value_, type }) => {
            console.log('### got clipboard value:\n', value_, type);
```


### Other Technical Details
<!--
Please provide the following information with your request and any other relevant technical details (versions of IDEs, local environment info, plugin information or links, etc).
-->

`npm --version` output: 9.6.6

`node --version` output: v18.16.0

`pod --version` output (iOS issues only): 1.12.1

### Additional Context
<!--
List any other information that is relevant to your issue. Stack traces, related issues, suggestions on how to fix, Stack Overflow links, forum links, etc.
-->

**Comment by apuchitnis** [comment]
@jcesarmobile -- just added a repro here! pls see the last commit: https://github.com/apuchitnis/ionic-team-capacitor-issues-6736

<img width="443" alt="image" src="https://github.com/ionic-team/capacitor/assets/10189197/fd268739-e1c5-46a0-b99b-db5df09e5c3c">

**Comment by jcesarmobile** [comment]
I can't reproduce in the provided app on an iOS 15.6 real device and iOS 16.4 simulator.

<img width="994" alt="Captura de pantalla 2023-07-24 a las 18 32 51" src="https://github.com/ionic-team/capacitor/assets/1637892/06eb0449-1a43-4681-9072-fa4d5bcb792a">

In which devices are you testing and how are you copying the text?
Does it paste correctly in other apps?

I copied the text using the copy button github provides in this own issue.

**Comment by apuchitnis** [comment]
hey @jcesarmobile - I was able to repro on an iPhone SE A2275 running iOS 16.5.1, and a simulator of iPhone 14 Pro running iOS 16.4

the pasting does seem to work correctly in other apps (eg whilst sending an SMS message) -- I get the same results as you do.

**Comment by jcesarmobile** [comment]
And which option are you choosing from the share menu on amazon?
If I click the "copy link" option I just get a short url, no text.

**Comment by apuchitnis** [comment]
I click `Copy` here (there is no "copy link"):
![IMG_6A478ED59A25-1](https://github.com/ionic-team/capacitor/assets/10189197/ccb9e631-196c-4460-b9e5-3501db1bda06)

**Comment by jcesarmobile** [comment]
Yeah, sorry, I was now testing on an iPad with iPadOS 17 and it's not showing a native dialog there like yours but a html dialog.
Testing on the iPhone I get the native dialog and I can see that the native clipboard has two strings and we only return one of them.

**Comment by apuchitnis** [comment]
Created a PR (linked above) to fix -- I'm new to Swift development, so lmk if this makes sense @jcesarmobile :)
